python/py-llvmlite-devel/Portfile

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim: fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+github.setup        numba llvmlite 409cd6d37fe1ba76438aa1a3a9c6a8adcd0ddb9d
+fetch.type          git
+name                py-llvmlite-devel
+conflicts           py310-llvmlite
+set my_name         py310-llvmlite
+revision            0
+categories-append   devel science
+license             BSD
+
+python.versions     310
+
+maintainers         {stromnov @stromnov} openmaintainer
+
+description         A lightweight LLVM python binding for writing JIT compilers
+long_description    ${description}
+
+homepage            https://llvmlite.pydata.org/
+
+checksums           rmd160  1c3a4f915657c5232ba123977a13827b2b1d2645 \
+                    sha256  26af66d1cad247da9554493848dc1c6b49ce427d702c624ad93ab2987d6adb29 \
+                    size    230091
+
+if {${name} ne ${subport}} {
+    patchfiles-append   patch-ffi_Makefile.osx.diff
+
+    set llvmver "11"
+
+    post-patch {
+        reinplace "s|%%CXX%%|clang++-mp-${llvmver}|" ${worksrcpath}/ffi/Makefile.osx
+
+        if {${os.major} <= 10} {
+            # https://trac.macports.org/ticket/61302
+            reinplace "s|%%MP_EXTRA_LDFLAGS%%|-framework CoreFoundation|" \
+                ${worksrcpath}/ffi/Makefile.osx
+        } else {
+            reinplace "s|%%MP_EXTRA_LDFLAGS%%||" ${worksrcpath}/ffi/Makefile.osx
+        }
+    }
+
+    depends_build-append \
+                        port:py${python.version}-setuptools \
+                        port:clang-${llvmver}
+
+    depends_lib-append  port:llvm-${llvmver} \
+                        port:zlib \
+                        port:ncurses \
+                        port:libxml2
+
+    build.env-append    LLVM_CONFIG=llvm-config-mp-${llvmver}
+    destroot.env-append LLVM_CONFIG=llvm-config-mp-${llvmver}
+} else {
+    github.livecheck.regex  {([0-9.]+)}
```

Comment:
this you probably don't want to set here: if you have a `-devel` port I would assume that you want to check for whatever update not only for released versions - right?